src/data_provider/src/sharedDefs.h

```diff
@@ -106,4 +108,24 @@ enum MacOSArchitecture
     ARM64
 };
 
+static const std::set<std::string> UNIX_PYPI_DEFAULT_BASE_DIRS
+{
+    "/usr/lib/python*/*-packages",
+    "/usr/lib64/python*/*-packages",
```

Comment:
Although the path is included I can not find this example package
![image](https://github.com/wazuh/wazuh/assets/13010397/56eb00ba-4431-46af-bb66-8e1e0cec8fcb)
![image](https://github.com/wazuh/wazuh/assets/13010397/56223d79-9694-4e17-8113-3a35a4b2421e)

Reply:
related to C++ standard definition.